pkg/dogstatsd/enrich.go

```diff
@@ -43,7 +43,8 @@ func enrichTags(tags []string, defaultHostname string, originTagsFunc func() []s
 	if entityIDValue == "" || !entityIDPrecedenceEnabled {
 		// Add origin tags only if the entity id tags is not provided
 		tags = append(tags, originTagsFunc()...)
-	} else if entityIDValue != entityIDIgnoreValue {
```

Comment:
Why not keep the `else if` with the `entityIDValue != ""` ?

Reply:
Not sure I follow, mind sketching it? The first `if` statement is with `||`.

Reply:
I missread the if statement, nevermind.

Reply:
Np, thank you.